xml/FrameworksIndex/netframework-3.5.xml

```diff
@@ -106309,18 +106309,6 @@
       <Member Id="M:System.Windows.Forms.AccessibleObject.Accessibility#IAccessible#accLocation(System.Int32@,System.Int32@,System.Int32@,System.Int32@,System.Object)" />
       <Member Id="M:System.Windows.Forms.AccessibleObject.Accessibility#IAccessible#accNavigate(System.Int32,System.Object)" />
       <Member Id="M:System.Windows.Forms.AccessibleObject.Accessibility#IAccessible#accSelect(System.Int32,System.Object)" />
-      <Member Id="M:System.Windows.Forms.AccessibleObject.Accessibility#IAccessible#get_accChild(System.Object)" />
```

Comment:
@joelmartinez Are these deletions expected for the mdoc preview?

Reply:
@gewarren ... yes, this one in particular, because it was the source of a bug (it's not an actual interface member, just a private compiler generated member)